app/controllers/coronavirus_form/session_controller.rb

```diff
@@ -3,6 +3,7 @@
 class CoronavirusForm::SessionController < ApplicationController
   def delete
     reset_session
-    redirect_to "/"
+    redirect_to I18n.t("leave_this_website.link_redirect_to") if params[:ext_r] == true
+    redirect_to "/" if params[:ext_r] != true
```

Comment:
Yeah, this looks good from a security perspective.

I'd probably suggest and if / else here, instead of the inline ifs.

Also I wonder if `/clear-session?leave-this-site=true` would be nicer than `/clear-session?ext_r=true`

Reply:
yeah i wondered about that, was thinking as this shows up in a users history, perhaps this is the one time where a less nice obscure param is ok?
I figured clear session is a bit of a nerd-term anyway?

Reply:
also turns out params are strings not booleans...